### DISCUSSION TITLE
Faulty reading on two digits

### DISCUSSION THREAD

**Discussion by cristianmitran** [jomjol/AI-on-the-edge-device#1157] [Q&A] [answered]
Hi,

Any idea how can I improve the faulty reading?
![image](https://user-images.githubusercontent.com/36613624/194768380-1952a94c-47c3-4cd4-a5ef-99487112c57a.png)

I have tried to use contrast and brightness but the first two digits are always faulty.
I know that the led light reflects a bit on the digits but the other digits at the right don't seem to be bothered.

Thanks,
Cristian

**Comment by ghost** [comment, marked as answer]
Do you tried different dig- models?

Edit: Tape the reflective areas. So you can raise the LED intensity.

**Reply by cristianmitran** [reply]
The option "dig-class11_1411_s2_q.tflite" did the job. Thank you.
